src/thunderbots/software/ai/hl/stp/evaluation/deflect_off_enemy_target.cpp

```diff
@@ -0,0 +1,67 @@
+//
+// Created by roark on 02/02/19.
+//
+
+#include "ai/world/field.h"
+#include "ai/world/world.h"
+#include "software/geom/line.h"
+#include "software/geom/util.h"
+
+auto MAX_RADIUS = 0.15;
+namespace Evaluation
+{
+    Point deflect_off_enemy_target(World world)
```

Comment:
I would prefer that we pass the parameters by const reference here, both for consistency with other functions and because it avoids needlessly copying the entire `World` object.
```suggestion
    Point deflect_off_enemy_target(const World& world)
```